gravitee-apim-console-webui/src/management/api/documentation-v4/documentation-pages-list/api-documentation-v4-pages-list.component.html

```diff
@@ -114,7 +114,7 @@
           <button mat-icon-button (click)="onMoveDown.emit(page)" [disabled]="i === dataSource.data.length - 1" aria-label="Move page down">
             <mat-icon svgIcon="gio:arrow-down"></mat-icon>
           </button>
-          <button mat-icon-button (click)="onDeletePage.emit(page.id)" disabled="true">
+          <button mat-icon-button (click)="onDeletePage.emit(page.id)" aria-label="Delete page">
```

Comment:
This button should be disabled if a folder is not empty or a page is used as general conditions.